### ISSUE TITLE
Build always fails once you set the CONTENT_ROOT to a wrong path

### ISSUE THREAD

**Issue by ugultopu** [mdn/yari#2638]
As discussed in https://github.com/mdn/yari/pull/2618#issuecomment-764831995:

# Steps to Reproduce
- `git clone git@github.com:mdn/yari.git`
- `cd yari && yarn && cd ..`
- `git clone git@github.com:mdn/content.git`
- `echo CONTENT_ROOT=$PWD/content > yari/.env` Note that `/files` is missing on purpose from `CONTENT_ROOT=$PWD/content` to simulate a user error.
- `cd yari && yarn dev` Build will succeed but content won't be there, since the wrong path for `CONTENT_ROOT` has been set. We realize this and try to fix the build now:
- `cd .. && echo CONTENT_ROOT=$PWD/content/files > yari/.env`
- `cd yari && yarn run` The build will fail with the following message even though `CONTENT_ROOT` is now correct (the following is an excerpt from the output of yarn dev. It is not a command that has been run by itself):

      $ node tool/cli.js gather-git-history --verbose
      Reusing existing history from <Path to your OS temprorary directory>/yari-git-history.json
      Error: ENOENT: no such file or directory, open '<Path to your working directory>/content/files/files/_githistory.json'
          at Object.openSync (node:fs:495:3)
          at Object.writeFileSync (node:fs:1519:35)
          at <Path to your working directory>/yari/tool/cli.js:303:12
          at Se._action (<Path to your working directory>/yari/tool/cli.js:27:13)
          at Se.run (<Path to your working directory>/yari/node_modules/@caporal/core/dist/index.js:1:27590)
          at processTicksAndRejections (node:internal/process/task_queues:93:5)
          at async Te._run (<Path to your working directory>/yari/node_modules/@caporal/core/dist/index.js:1:32257)

# Expected Results
The build will not fail once the `CONTENT_ROOT` is set to the correct value.

# Workaround
Remove the file at `<Path to your OS temprorary directory>/yari-git-history.json` manually and attempt the build again.

# Further Thoughts
The build is also writing a file named `_githistory.json` under `content/files`. In the first instance (the one with the wrong `CONTENT_ROOT`), the path to this file is `<Path to MDN content>/files/_githistory.json`. In the second instance (the one with the correct `CONTENT_ROOT`), the path to this file is `<Path to MDN content>/files/en-us/_githistory.json`. These files seem to have the exact same content with the file at `<Path to your OS temprorary directory>/yari-git-history.json`. So what's the purpose of creating two files with the same content?

**Comment by ddbeck** [comment]
Possibly a duplicate of #2452 (though these reproduction steps are better 😅)

**Comment by peterbe** [comment]
I believe this is happening because of a fix that hasn't landed yet: https://github.com/mdn/yari/pull/2661
Once that lands it's going to be a lot more forgiving if the `CONTENT_ROOT` doesn't end with `files` when it should.
Let's see if that fixes it next week.

**Comment by peterbe** [comment]
Given https://github.com/mdn/yari/pull/2661#issuecomment-765703823 can we close this issue now?
